native/dll/DShowCapture-2/GraphManager.cpp

```diff
@@ -60,6 +60,7 @@ static char* BDATypeCapString( DWORD BDACap );
 int BDATypeNum( DWORD dwBDACap );
 static char* GetBDAType( DWORD dwBDACap, int nIndex );
 void SwitchBDAConnect( DShowCaptureInfo* pCapInfo );
+long int DETECTED_CAPTURE_NUM;
```

Comment:
Remove this line, this is creating a global variable which is bad style and it will get overridden if multiple capture graphs are instantiated in parallel. See below for what to do instead.

Reply:
Done